data/mods/PKs_Rebalance/pk_critter_factions.json

```diff
@@ -12,6 +12,31 @@
         "friendly" : ["blob", "cult"],
         "neutral" : ["small_animal", "insect"],
         "by_mood" : ["ant"]
+    },{
+        "type" : "MONSTER_FACTION",
+        "name" : "doom",
+        "neutral" : ["vermin"],
+        "by_mood" : ["small_animal"]
```

Comment:
Everything is by default "by_mood" to everything unspecified. Meaning you only need to add it if a lower faction overrides it somehow.